### PR TITLE
Added missing configuration entries

### PR DESCRIPTION
**Pull request by junquero**
The two configuration entries forceDeleteTenantAllowed and forceDeleteNamespaceAllowed exist in broker.conf, but they were missing in standalone.conf, so they were not able to be configured by environment variables.

### CODE REVIEW

**Review by codelipenghui — APPROVED** [review]

**Comment by junquero** [comment]
The CI - Unit has failed, but I don't think this minor change has nothing to do with that fail.
Also I don't know if I have to wait for a second approval to have the merge option enabled.

**Pull request merged**
